include/os_lib.h

```diff
@@ -12,4 +12,4 @@
  * call_parameters[2+] = called function parameters
  */
 SYSCALL void os_lib_call(unsigned int *call_parameters PLENGTH(3*sizeof(unsigned int)));
-SYSCALL void os_lib_end(void);
+SYSCALL void __attribute__((noreturn)) os_lib_end(void);
```

Comment:
Maybe this should be under HAVE_BOLOS ifdef has here: https://github.com/LedgerHQ/ledger-secure-sdk/blob/master/include/os_task.h#L31-L35

Reply:
Done